### PR TITLE
Add hood pot reset on robotInit

### LINKED ISSUE #67: Reset Hood Pot On Robot Init

### PR DESCRIPTION
**Pull request by Jacob4649**
Fix #67

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Hood.java [hunk 1]
```diff
@@ -20,6 +20,8 @@
   private final Servo m_servo;
   private final AnalogInput m_pot;
 
+  private double m_bottomValue;
```
Comment (matthew-reynolds) [inline src/main/java/frc/robot/subsystems/Hood.java]: `m_minRawAngle`

File: src/main/java/frc/robot/subsystems/Hood.java [hunk 2]
```diff
@@ -58,8 +60,17 @@ public boolean atSetpoint() {
    * @return the angle of the hood in degrees
    */
   public double getAngle() {
-    return (((m_pot.getVoltage() / Constants.AIO_MAX_VOLTAGE) * Constants.Hood.POT_SCALE) - Constants.Hood.POT_OFFSET)
-        * (Constants.Hood.MAX_ANGLE - Constants.Hood.MIN_ANGLE) + Constants.Hood.MIN_ANGLE;
+    return (getRawAngle() - m_bottomValue) * (Constants.Hood.MAX_ANGLE - Constants.Hood.MIN_ANGLE)
+        + Constants.Hood.MIN_ANGLE;
```
Comment (matthew-reynolds) [inline src/main/java/frc/robot/subsystems/Hood.java]: Divide by raw expected range (Should be ~360deg)

File: src/main/java/frc/robot/subsystems/Hood.java [hunk 3]
```diff
@@ -86,13 +97,21 @@ public double getSetpoint() {
    * @param speed the speed to set the servo to as a percent from -1 to 1
    */
   public void set(double speed) {
-    if ((getAngle() < Constants.Hood.MIN_ANGLE && speed < 0) || (getAngle() > Constants.Hood.MAX_ANGLE && speed > 0)) {
+    if ((getRawAngle() < Constants.Hood.MIN_RAW_ANGLE && speed < 0)
+        || (getRawAngle() > Constants.Hood.MAX_RAW_ANGLE && speed > 0)) {
```
Comment (matthew-reynolds) [inline src/main/java/frc/robot/subsystems/Hood.java]: Use hood hard stops rather than pot hard stops

**Review by matthew-reynolds — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Hood.java (hunk 3 above)
Comment (Jacob4649) [reply src/main/java/frc/robot/subsystems/Hood.java]: resolved in acedb0d

File: src/main/java/frc/robot/subsystems/Hood.java (hunk 2 above)
Comment (Jacob4649) [reply src/main/java/frc/robot/subsystems/Hood.java]: resolved in acedb0d

File: src/main/java/frc/robot/subsystems/Hood.java (hunk 1 above)
Comment (Jacob4649) [reply src/main/java/frc/robot/subsystems/Hood.java]: resolved in acedb0d

**Review by matthew-reynolds — APPROVED** [review]

**Pull request merged**